src/components/TextInput/_TextInput.scss

```diff
@@ -0,0 +1,75 @@
+$break-input-mobile: 400px;
+
+.textinput {
+  .inputfield {
+    @include space-inset-xs;
+    @include space-stack-xs;
+
+    border: 1px solid var(--ui-gray-medium);
+
+    &[type="text"],
+    &[type="number"],
+    &[type="password"],
+    &[type="url"],
+    &[type="tel"],
+    &[type="email"] {
+      width: 100%;
+    }
+
+    &--textarea {
+      min-height: var(--space-xxl);
+      width: 100%;
+    }
+
+    &::-webkit-input-placeholder {
+      color: var(--ui-gray-dark);
+    }
+
+    &:-moz-placeholder {
+      color: var(--ui-gray-dark);
+    }
+
+    &:-ms-input-placeholder {
+      color: var(--ui-gray-dark);
+    }
+
+    &:hover {
+      border-color: var(--ui-gray-dark);
+    }
+
+    &:disabled {
+      background-color: var(--ui-gray-xxlight);
+      color: var(--ui-gray-xdark);
+      pointer-events: none;
+    }
+
+    &:active,
+    &:focus,
+    &:active:hover,
+    &:focus:hover {
+      border: 1px solid var(--ui-focus);
+      box-shadow: 0;
+      outline: 1px solid var(--ui-focus);
+      z-index: 9999;
+    }
+
+    &--error {
+      border: 1px solid var(--ui-error);
+    }
+  }
+
+  label {
+    font-weight: 300;
+  }
+}
+
+.input-group {
+  @include breakpoint($break-input-mobile) {
+    display: flex;
+    justify-content: space-between;
+
+    > *:not(:last-child) {
+      @include space-inline-m;
+    }
+  }
+}
```

Comment:
Will there be an input group component in the same way we are planning to do a checkbox and radio group? If so, these styles would probably belong in the group component instead.

Reply:
@kaplanmichael 
The `Input Group` story was part of the original Input component and I simply kept it along for the ride as I was working on this component.  I actually don't think it is necessary, so I will remove it.  And with that, I do not think a `TextInputGroup` component is necessary.  Do you?

Reply:
Yeah, probably not.  Most people would use some kind of grid or other layout tool to organize their text inputs.  With checkboxes and radios, a component that groups them and encapsulates their logic is probably more helpful.